src/io/flutter/run/daemon/DevToolsService.java

```diff
@@ -71,6 +72,17 @@ private DevToolsService(@NotNull final Project project) {
     });
   }
 
+  public DevToolsInstance getDevToolsInstance() {
```

Comment:
expose the Future directly instead of providing a method that might lock the UI thread.

Reply:
I originally wanted this function to manage restarting if the daemon command fails, but that can be done elsewhere too